### ISSUE TITLE
Sporadic warning "Extension sourcegraph/$LANG was not found"

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#10558]
This gets spammed into the console every once in a while:

<img width="663" alt="Screen Shot 2020-05-06 at 21 07 12" src="https://user-images.githubusercontent.com/10532611/81562453-2b0e7780-9395-11ea-8f00-5970a2150986.png">

Iirc they don't work in that case, and require a page reload.

It seems to be some kind of race condition.

This may be easy to solve as part of [RFC 155 Extension API Architecture](https://docs.google.com/document/d/1ikrUNVe3YVbR-JpegxhjrFdmRkTGzTLcOMkKHnOyjuE/edit#heading=h.trqab8y0kufp).

**Comment by Joelkw** [comment]
@felixfbecker is this still a problem or was it solved in our RFC 155 work?

**Comment by felixfbecker** [comment]
I don't think this would be solved yet because we haven't reached extensions activation in RFC 155 work yet.

**Comment by felixfbecker** [comment]
(it's also hard to verify though because it's a rare race condtion)
